src/pooler.erl

```diff
@@ -392,8 +392,10 @@ handle_info(timeout, #pool{group = Group} = Pool) ->
 handle_info({'DOWN', MRef, process, Pid, Reason}, State) ->
     State1 =
         case dict:find(Pid, State#pool.all_members) of
+            %% the connection is not alive
```

Comment:
This comment is a bit off.  This case is when the process that has gone down is a known member of the pool